subjects/java/checkpoints/universal-greeting/README.md

```diff
@@ -0,0 +1,42 @@
+## UniversalGreeting
+
+### Instructions
+
+In a file named `UniversalGreeting.java` write a function `greeting` that takes a String language as a parameter and returns a greeting message based on the language as follows:
+
+`FR` : `Bonjour comment allez-vous?`
+`EN`: `Hello, How are you?`
+`ES` : `Hola, cómo estás?`
+
+and return an empty string in any other case.
+
+### Expected Functions
+
+```java
+public class UniversalGreeting {
+    public static String greeting(String message) {
+        // your code here
+    }
+}
+```
+
+### Usage
+
+Here is a possible ExerciseRunner.java to test your function :
```

Comment:
I would keep the same formatting style for the file name `ExerciseRunner.java`

Reply:
done